README.md

```diff
@@ -33,3 +40,4 @@ For simple starter examples, see the
 ## Contributing to Config Connector
 
 Please refer to our [contribution guide](CONTRIBUTING.md) for more details.
```

Comment:
Is this link broken now?

Reply:
Good catch. I put back the CONTRIBUTION.md. I think it does not contain much TF/DCL specific content